.travis.yml

```diff
@@ -33,9 +33,14 @@ before_install:
   - conda config --set always_yes yes
   - conda install conda-build anaconda-client
   - conda update -q conda conda-build
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION pyqt=$PYQT_VERSION pip numpy scipy six psutil pyqtgraph -c conda-forge
+  # Add the channels needed for install
+  - conda config --append channels conda-forge
+  - conda config --append channels gsecars
```

Comment:
I just noticed that `pyepics` is now available through `conda-forge` so we can deprecate the usage of `gsecars`.

Reply:
https://anaconda.org/conda-forge/pyepics